### PR TITLE
Link shortcut and formatting options zindex fix

### PR DESCRIPTION
**Pull request by thecalcc**
SDNTB-832

### CODE REVIEW

File: scripts/apps/workspace/content/views/FormattingOptionsMultiSelect.tsx [hunk 1]
```diff
@@ -50,6 +50,7 @@ export class FormattingOptionsTreeSelect extends React.Component<IProps> {
                 label={gettext('Formatting options')}
                 labelHidden
                 inlineLabel
+                zIndex={2000}
```
Comment (tomaskikutis) [inline scripts/apps/workspace/content/views/FormattingOptionsMultiSelect.tsx:53]: do 1050 or next one that works instead of jumping a thousand

**Review by tomaskikutis — APPROVED** [review]
verify also in the UI that after keybinding gets hit - the keybinding focus goes to link input

**Comment by thecalcc** [comment]
> verify also in the UI that after keybinding gets hit - the keybinding focus goes to link input
@tomaskikutis 
That's not the case even if you click on it with your mouse. This is different because the style of this formatting option is different. Instead it's indicated that this is the formatting option which is active by the popup being displayed.

**Comment by thecalcc** [comment]
@tomaskikutis you'll notice I added the `&& selectionState.isCollapsed() !== true` check. That's because we have to "disable" the possibility to activate the link option if nothing is selected. This is basically mimicking the normal functionality (when clicking on it with your mouse). If there's no text selected we disable the option to add a link.

**Pull request merged**